docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: "Introduction to the Dagster Tutorial and Assets | Dagster Docs"
+description: "Learn about the objectives for the Dagster tutorial and about assets"
+noindex: True
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+
+In this tutorial, you will analyze activity on a popular news aggregation website. Similar to your data pipelines, you will fetch data from the website, clean it up, and build a report that summarizes some findings. You'll then tell Dagster to occassionally update the data and the report, which Dagster calls _assets_.
+
+---
+
+## Introduction to Software-defined assets
+
+Before building a Dagster project, you'll first learn about Dagster's core concept: the Software-defined asset (SDA).
+
+An asset is an object in persistent storage that captures some understanding of the world. Assets can be any type of object, such as:
+
+- A database table or view
+- A file, such as in your local machine or blob storage like Amazon S3
+- A machine learning model
+
+A Software-defined asset is Dagster's concept that couples an asset to the function and upstream assets that are used to produce its contents.
+
+If you have an existing data pipeline, you likely already have assets.
+
+### Goals
+
+By the end of this section, you will:
+
+- Know what assets are
+- Know the power of Software-defined assets
+- Know when to use task-based orchestration with assets
+- Start to think about what parts of your current pipeline are assets
+
+### Parts of a Software-defined asset
+
+Software-defined assets wrap objects to enable data engineers to do more with them.
```

Comment:
if you decide to address the benefits of SDAs a little bit above, we can eliminate this line